metabase/metabase
# feat(sdk): expose color and typography options for smart scalar in embedding SDK

Closes https://github.com/metabase/metabase/issues/42563

### Description

Expose the positive and negative colors used in the previous value comparison to the theme settings, as well as the scalar value's font size and line height options.

### How to verify

- Navigate to the `test-sdk-scalar` branch in the demo app
- Note the values defined in `AppProvider.tsx` - you can change the colors and font size for testing.
- Verify that the CSS workarounds in `product-list.css` are deleted
- Go to the product page in the demo. The positive color and value font size/line height should be updated accordingly.

### Demo

<img src="https://github.com/metabase/metabase/assets/4714175/1c1d8449-ae1c-4388-b1d8-ba7915276337" width="300">
